### PR TITLE
Request to merge "HTML: SIGN UP PAGE" to master

### PR DESCRIPTION
**Pull request by RailonA**
We made a clone of the Intuit sign up page. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has a mock sign-up form with labels for email, phone number, and password.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  flex-direction: row;
+  justify-content: center;
+  margin: 8px;
+}
+
+.footer-div p {
+  font-size: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
```
Comment (Temmarie) [inline styles.css]: Your footer needs a little tweak in order to look like the actual site's. 
```suggestion
}footer {
    display: flex;
    flex-direction: column;
    width: 100%;
    margin-top: 24px;
}

.footer-div {
    display: flex;
    flex-direction: row;
    justify-content: center;
    /* margin: 8px; */
    margin-bottom: 5px;
}

.footer-div p {
    font-size: 12px;
    /* margin-left: 5px; */
    /* margin-right: 5px; */
    line-height: 0;
    color: rgb(80, 80, 80);
    font-weight: 500;
    font-family: sans-serif;
}

```

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+  margin: 10px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: white;
+  width: 380px;
+  height: 600px;
```
Comment (Temmarie) [inline styles.css]: Increase the form height in order to get enough space for the Privacy statement.

```suggestion
  height: 618px;
```

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
### Hello Team! 
 
### You have done well so far :clap: However, there are some changes you have to make for your project to be approved. Kindly work on them and submit them for another code review.



**Recommended**

- [ ] The font-size used in the "top-nav"  which has the sign in button is too big.  
- [ ] The intuits products row should all be links (PS: it should be an image and a text beside it).
- [ ] Fix your header to look more like the [actual site](https://accounts.intuit.com/signup.html), you have some texts that are not in the original Intuit form.
- [ ] The form labels font-size is bigger than that of the actual site.
- [ ] The Privacy statement and the recaptcha tabs are in need of a little bit of work in order to get them to look as similar as possible to the actual site.
- [ ] The footer and copyright needs some work as well. Consider the comments I've left below to help you fix those issues.
- [ ] The "Create account" button should have a type of "submit". 


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review @tamara._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by ElijahTobs — APPROVED** [review]
# Status: Approved! :tada: 

Congratulations! @RailonA and @DsunilK 

### Good job implementing the changes requested by the previous TSE.

## You can now merge and submit.

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Pull request merged**